Gaijinizer patch/CommonEvents_112.txt

```diff
@@ -32,28 +32,31 @@ Fuck... This!!#afrg
 エンプーサは思わず\\N[1]の拘束を解いた！！！
 > CONTEXT: CommonEvents_112/19/Dialogue
 > CONTEXT: CommonEvents_112/53/Dialogue
-Empuse involuntarily solved \\N[1]'s bound!!!#MTLed
+Empuse has released \\N[1]!!!#Ron
 > END STRING
 
 > BEGIN STRING
 エンプーサは反動で\\C[2]体勢を崩しているようだ…！
 > CONTEXT: CommonEvents_112/24/Dialogue
-Empuse is in reaction\\C[2] It seems to be disorganizing ...!#MTLed
+Empuse's recoiling, \\C[2] it looks like she's been
+thrown off balance...!#Ron
 > END STRING
 
 > BEGIN STRING
 あーあ、エンプちゃん怒らせちゃった…
 
 わたししーらないっと
 > CONTEXT: CommonEvents_112/57/Dialogue/enemy2
-Ah, ah, Empuse-chan I got angry ...
-I'm not sorry#MTLed
+*sigh* Empuse-chan's mad now...
+
+Don't come crying to me.#Ron
 > END STRING
 
 > BEGIN STRING
 エンプーサは蹴られた反動で\\C[2]体勢を崩しているようだ…！
 > CONTEXT: CommonEvents_112/62/Dialogue
-Empuse is kicked back\\C[2] It seems to be disorganizing ...!#MTLed
+Empuse is recoiling from being kicked, \\C[2] it looks
+like she's been thrown off balance...!#Ron
 > END STRING
```

Comment:
Rephrase this for flow:
>_**Empuse is recoiling from being kicked....
\\\C[2]it looks like she's been thrown off balance...!**_